### PR TITLE
Pro features should not be available in Pila machines

### PR DESCRIPTION
**Pull request by celine-m-s**
Trello ticket # 66: https://trello.com/c/H1r2OwhM/66-filtrer-les-bornes-pila-sur-la-version-pro

### CODE REVIEW

**Comment by vgrange** [comment]
ticket https://trello.com/c/H1r2OwhM/66-filtrer-les-bornes-pila-sur-la-version-pro

File: labonneboite/tests/web/front/test_pro_version.py [hunk 1]
```diff
@@ -1,55 +1,77 @@
...
 class ProVersionTest(DatabaseTest):
 
-    @mock.patch('labonneboite.conf.settings.VERSION_PRO_ALLOWED_EMAIL_SUFFIXES', ['@pole-emploi.fr'])
+    def setUp(self):
+        super().setUp()
+        self.pro_user = User.create(
+            email='john.doe@pole-emploi.fr',
+            gender='male',
+            first_name='John',
+            last_name='Doe'
```
Comment (vgrange) [inline labonneboite/tests/web/front/test_pro_version.py]: pinaillage: trailing comma on last list item (x3)

**Comment by vgrange** [comment]
[Filtrer les bornes PILA sur la version PRO](https://trello.com/c/H1r2OwhM/66-filtrer-les-bornes-pila-sur-la-version-pro)

File: labonneboite/tests/web/front/test_pro_version.py (hunk 1 above)
Comment (celine-m-s) [reply labonneboite/tests/web/front/test_pro_version.py]: Ajouté !

**Pull request merged**